hackr/crypto.py

```diff
@@ -62,7 +64,7 @@ def bcrypt(self):
             f = open(self.wordlist, "r")
             for i in f:
                 plain_text = i.rstrip('\n')
-                if bcrypt.checkpw(plain_text, self.hash):
+                if bcrypt.checkpw(plain_text.encode('utf-8'), self.hash):
```

Comment:
particular you added this?
is it particular to `python3`?

Reply:
Actually, I have made changes in almost all the files.
Yes, the changes are specific to Python3.
Yes, I have covered all the modules,I ran all the commands you have mentioned in the ReadMe and all worked fine and also tested functions individually.
If something breaks, please let me know.